src/GoodsList.tsx

```diff
@@ -1,16 +1,25 @@
 import React from 'react';
 import { Good } from './types/Good';
 
-type Props = {
+type PropsGoodList = {
   goods: Good[]
 };
 
-export const GoodsList: React.FC<Props> = ({ goods }) => (
+export const GoodsList: React.FC<PropsGoodList> = ({ goods }) => (
   <ul>
-    {goods.map(good => (
-      <li key={good.id} data-cy="good">
-        {good.name}
-      </li>
-    ))}
+    {goods.map(good => {
+      const { id, name, color } = good;
+      const colorStyle = { color: `${color}` };
```

Comment:
It's the same as color: color
```suggestion
      const colorStyle = { color };
```

Reply:
https://prnt.sc/G5bfLuX5rp25

Reply:
It's because you should do it without additional brackets. `style = { color }` instead of `style = { color: { color}`

Reply:
.